tests/unit/keyspace.tcl

```diff
@@ -306,7 +306,11 @@ start_server {tags {"keyspace"}} {
     test {COPY basic usage for ziplist hash} {
         r del hash1 newhash1
         r hset hash1 tmp 17179869184
-        assert_encoding listpack hash1
+        if {$::packed_encoding != ""} {
+            assert_encoding $::packed_encoding hash1
+        } else {
+            assert_encoding listpack hash1
+        }
```

Comment:
I tried adding ``config get default-packed-encoding``, but found that this change would cause the introspection test to fail, it would first ``config get *``, then ``config set``, but ``default-packed-encoding`` would not have  ``` config set ``` .
So I'm asserting_packed_encoding to determine if it's listpack or ziplist.